app/controllers/application_controller.rb

```diff
@@ -16,6 +16,8 @@ class ApplicationController < ActionController::API
     Breakers::OutageException
   ].freeze
 
+  EXCEPTIONS_WITH_RECORD_NOT_FOUND = SKIP_SENTRY_EXCEPTION_TYPES + [Common::Exceptions::RecordNotFound]
```

Comment:
Related to @saneshark's comment about inheritance, I think this is a case where you might want composition instead of inheritance. Some controllers want to ignore record not found, but not all, so it could go in a module rather than adding new constant in the base class and overriding `skip_sentry_exception_types` in each subclass:

``` ruby
module IgnoreNotFound
  def skip_sentry_exception_types
    ApplicationController::SKIP_SENTRY_EXCEPTION_TYPES + [Common::Exceptions::RecordNotFound]
  end
end

class EVSSClaimsController
  include IgnoreNotFound
end
```